BibBuddy/app/src/main/java/de/bibbuddy/ExportBibTex.java

```diff
@@ -181,16 +181,19 @@ private String getBibKey(Book book) {
 
   private String getBibNotesFromBook(Book book, NoteDao noteDao) {
     List<Long> notesList = noteDao.getAllNoteIdsForBook(book.getId());
+    NoteTypeLut noteType = NoteTypeLut.TEXT;
     StringBuilder allNotes = new StringBuilder();
 
-    if (!notesList.isEmpty()) {
+    if (!notesList.isEmpty() && noteType.getId() == 1) {
```

Comment:
This is always true: noteType.getId() == 1. So it's redundant. You didn't make it dependent on a note item.
```suggestion
    if (!notesList.isEmpty()) {
```

Reply:
What is the difference with the previous comment?